frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/state-update/storage-tab-state-update.ts

```diff
@@ -16,13 +16,24 @@ import { DiskWrapper } from '../../../../k8s/wrapper/vm/disk-wrapper';
 
 export const prefillInitialDiskUpdater = ({ id, prevState, dispatch, getState }: UpdateOptions) => {
   const state = getState();
-  if (!hasVmSettingsChanged(prevState, state, id, VMSettingsField.PROVISION_SOURCE_TYPE)) {
+  const baseDiskImageChanged =
+    (hasVmSettingsChanged(prevState, state, id, VMSettingsField.OPERATING_SYSTEM) ||
+      hasVmSettingsChanged(prevState, state, id, VMSettingsField.FLAVOR) ||
```

Comment:
these can be verified in one call instead of multiple ones

Reply:
nice, done :+1: